config/samples/resources/managedkafkatopic/managedkafka_v1beta1_managedkafkatopic.yaml

```diff
@@ -0,0 +1,29 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: managedkafka.cnrm.cloud.google.com/v1beta1
+kind: ManagedKafkaTopic
+metadata:
+  name: managedkafkatopic-sample
+spec:
+  projectRef:
+    # Replace ${PROJECT_ID?} with your project ID
+    external: ${PROJECT_ID?}
+  location: us-central1
+  clusterRef:
+    name: managedkafkatopic-dep
+  partitionCount: 2
+  replicationFactor: 3
```

Comment:
Does MKT give default value to these two fields?

Reply:
Both fields are required.

Reply:
Marked both fields as "required" in https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/3742/commits/bff9ffac6a6c3263cb1df6b3c72813ed60eab157